packages/navigator/src/browser/AbstractNavigatorTreeWidget.ts

```diff
@@ -0,0 +1,67 @@
+// *****************************************************************************
+// Copyright (C) 2017 TypeFox and others.
```

Comment:
We work at Ericsson and are in 2022 ;)

```suggestion
// Copyright (C) 2022 Ericsson and others.
```